relnotes/timedivision.deprecation.md

```diff
@@ -0,0 +1,5 @@
+### Round relative time interval arguments
```

Comment:
The commit message for "Rename isTimeInterval to isRelativeTime" should mention the reason for this change. (Currently, while the code change looks fine, there's no context as to why we might want to merge this change.)